torchtune/modules/loss/kd_losses.py

```diff
@@ -137,4 +137,7 @@ def forward(
                 student_chunk, teacher_chunk, label_chunk, normalize=False
             )
 
+        if torch.sum(mask.view(-1), dim=0) == 0:
```

Comment:
Let's pull this out into a reasonably named variable.

Reply:
updated.